galaxy_ng/app/api/ui/serializers/user.py

```diff
@@ -0,0 +1,80 @@
+from rest_framework import serializers
+from rest_framework.exceptions import ValidationError
+from galaxy_ng.app.api import permissions
+
+from galaxy_ng.app.models import auth as auth_models
+
+
+class GroupSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = auth_models.Group
+        fields = (
+            'id',
+            'name'
+        )
+
+
+class UserSerializer(serializers.ModelSerializer):
+    password_confirm = serializers.CharField(write_only=True, allow_blank=True, required=False)
+
+    class Meta:
+        model = auth_models.User
+        fields = (
+            'id',
+            'username',
+            'first_name',
+            'last_name',
+            'email',
+            'groups',
+            'password',
+            'password_confirm'
+        )
+        extra_kwargs = {
+            'password': {'write_only': True, 'allow_blank': True, 'required': False}
+        }
+
+    def validate_password(self, password):
+        if not password:
+            return
+        if len(password) <= 10:
+            raise ValidationError(detail={
+                'password': "Password must be 10+ characters long"})
```

Comment:
Error message looks inconsistent with actual check. Does `10+` mean range 10 and higher (including 10 itself)?
This code will fail if length of the password is exactly 10 characters.